safepoint-analyzer-commons/src/test/groovy/pl/ks/profiling/safepoint/analyzer/commons/shared/gc/parser/GCUnifiedLogFileParserSpec.groovy

```diff
@@ -245,8 +245,8 @@ class GCUnifiedLogFileParserSpec extends Specification {
         gcEntry.regionsMax[REGIONS_SURVIVOR] == 184
         gcEntry.regionsMax[REGIONS_OLD] == null
         gcEntry.regionsMax[REGIONS_HUMONGOUS] == null
-        gcEntry.regionsSizeAfterGC == null
```

Comment:
I'm not sure if `[:]` is better than `Map.of()`. Let me know if you want me to change it.

Reply:
```Collections.emptyMap()```?